connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/executable/BatchExecutableProcessor.java

```diff
@@ -191,8 +191,8 @@ private RegisteredExecutable activateSingle(
         data.deduplicationId());
 
     if (metricsRecorder != null) {
-      metricsRecorder.increase(
-          Inbound.METRIC_NAME_ACTIVATIONS, Inbound.ACTION_ACTIVATED, data.type());
+      var type = data.type() + "#" + data.connectorElements().getFirst().element().version();
```

Comment:
@mathias-vandaele I dont think this will create correct tags. AFAIK we have to create a custom metrics recorder as discussed that tracks the `elementTemplateId` & `elementTemplateVersion` in separate tags when recording the count. The `type` should also be a separate tag.